src/app/pages/users/user/settings/activity/activity-card/activity-card.component.ts

```diff
@@ -1,8 +1,7 @@
-import { Component, Input, ViewChild } from '@angular/core';
+import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/HEPIA-Association/association-WebClient/pullRequest?prid=7914646)
```suggestion
import { FormControl } from "@angular/forms";
```